src/main/java/frc/robot/constants/Constants2024.java

```diff
@@ -6,10 +6,10 @@
 public class Constants2024 extends GameConstants {
     //AMP 
     public static final int AMP_ID = 5;
-    
 
     //RAMP
     public static final double RAMP_ERROR_RANGE = 0.00;
+    public static final double RAMP_ANGLE = 15.0;
```

Comment:
This is not used and was deleted from main

Reply:
done